specification.md

```diff
@@ -88,6 +88,13 @@
     - [ ] Construct a job that uses all the various "knobs" of the resource
     and job specifications (with some verbose comments thrown in)
 
+<<<<<<< HEAD
```

Comment:
This looks conflicted...

Reply:
Not sure what happened there. Should be fixed now.